rpc/handlers.go

```diff
@@ -38,7 +38,10 @@ var (
 	ErrClassAlreadyDeclared       = &jsonrpc.Error{Code: 51, Message: "Class already declared"}
 	ErrInternal                   = &jsonrpc.Error{Code: jsonrpc.InternalError, Message: "Internal error"}
 	ErrInvalidTransactionNonce    = &jsonrpc.Error{Code: 52, Message: "Invalid transaction nonce"}
-	ErrInsufficientMaxFee         = &jsonrpc.Error{Code: 53, Message: "Max fee is smaller than the validation's actual fee"}
+	ErrInsufficientMaxFee         = &jsonrpc.Error{
```

Comment:
let's inline it and if linter complains about `lll` then ignore it

Reply:
done